src/js_printer.zig

```diff
@@ -2039,7 +2033,9 @@ fn NewPrinter(
                     p.print(".require(");
                     {
                         const path = input_files[record.source_index.get()].path;
-                        p.printInlinedEnum(.{ .number = @floatFromInt(path.hashForKit()) }, path.pretty, level);
+                        p.print('"');
```

Comment:
what's the story here?

Reply:
using numbers for module specifiers is going to be too much of a debugging burden on myself and framework developers. it isn't worth the unproven performance boost this idea had, especially considering the two other bundlers on my mind see decent performance with strings (and their string contents are much more verbose).

if E.String is too big then that should be optimized